File: javascript/examples/FeatureCollection/CountFeatures.js
// Count Features
// #section FeatureCollection:8
//
// Count Panoramio photos near SF that mention bridges.

var photosNearSF = ee.FeatureCollection(
    'ft:1qpKIcYQMBsXLA9RLWCaV9D0Hus2cMQHhI-ViKHo');
var brigdePhotos = photosNearSF.filter(ee.Filter.or(
    ee.Filter.contains('title', 'Bridge'),
    ee.Filter.contains('title', 'bridge')));

addToMap(photosNearSF, {color: '0040b0'});
addToMap(brigdePhotos, {color: 'e02070'});
centerMap(-122.39, 37.7857, 12);

alert('There are ' +
      brigdePhotos.aggregate_count('.all') +
      ' bridge photos around SF.');

File: python/examples/py/FeatureCollection/count_features.py
"""Count features example.

Count Panoramio photos near SF that mention bridges.
"""
import ee
import ee.mapclient

ee.Initialize()
ee.mapclient.centerMap(-122.39, 37.7857, 12)

photos_near_sf = ee.FeatureCollection('GOOGLE/EE/DEMOS/sf-photo-locations')
bridge_photos = photos_near_sf.filter(
    ee.Filter().Or(ee.Filter.stringContains('title', 'Bridge'),
                   ee.Filter.stringContains('title', 'bridge')))

ee.mapclient.addToMap(photos_near_sf, {'color': '0040b0'})
ee.mapclient.addToMap(bridge_photos, {'color': 'e02070'})

print('There are {} bridge photos around SF.'
      .format(bridge_photos.size().getInfo()))
